frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -5,28 +5,22 @@
 }
 
 .add-capacity-modal--padding {
-  .co-storage-class-dropdown {
-    .dropdown {
-      width: 21.5rem;
-        button {
-          width: 100%;
-        }
-    }
-  }
-  .form-group input {
-    max-width: 17rem;
-    margin-right: 5px;
-  }
-  .toolTip_dropdown {
-    button {
-      position: absolute;
-      left: 6.6rem;
-      top: 8.5rem;
-    }
-  }
   padding-top: 2em;
 }
 
+.add-capacity-modal--padding .co-storage-class-dropdown .dropdown {
+  width: 21.5rem;
+}
+
+.add-capacity-modal--padding .co-storage-class-dropdown .dropdown button {
+  width: 100%;
+}
+
+.add-capacity-modal--padding .form-group input {
```

Comment:
Give this a class name that follows BEM naming.

Reply:
ack